src/app/components/shared/header/header.component.ts

```diff
@@ -36,13 +82,8 @@ export class HeaderComponent implements OnInit {
         label: 'My Dashboard',
         routerLink: ['/myDashboard'],
       },
-      {
-        label: 'Sing in',
-        icon: 'pi pi-fw pi-sign-in',
-        command: () => {
-          const dialogRef = this.dialog.open(SingInComponent);
-        },
-      },
     ];
+    this.items.push(this.singInBtn);
```

Comment:
You can avoid pushing the signInMenuItem. Just set is as a last object of the array

Reply:
Fixed